### ISSUE TITLE
Add support for scoring metrics to Runtime.cross_validate()

### ISSUE THREAD

**Issue by raghukiran1224** [project-codeflare/codeflare#21]
## Overview
Current implementation does not accept user specified scoring metric(s). For example, 
`cross_val_score(pipeline, X_test, y_test, scoring="neg_mean_squared_error", cv=10)`
The list of sklearn model evaluation metrics are listed here 
https://scikit-learn.org/stable/modules/model_evaluation.html#scoring-parameter

### Acceptance Criteria
One common use case documented below is supported by codeflare pipeline. 
https://scikit-learn.org/stable/auto_examples/model_selection/plot_underfitting_overfitting.html#sphx-glr-auto-examples-model-selection-plot-underfitting-overfitting-py

**Comment by raghukiran1224** [comment]
created by @yuanchi2807
